pkg/config/lima_config_applier_test.go

```diff
@@ -81,6 +85,34 @@ func TestDiskLimaConfigApplier_Apply(t *testing.T) {
 				&yaml.TypeError{Errors: []string{"line 1: cannot unmarshal !!str `this is...` into limayaml.LimaYAML"}},
 			),
 		},
+		{
+			name: "lima config file with additional directories",
+			config: &Finch{
+				Memory:                pointer.String("2GiB"),
+				CPUs:                  pointer.Int(4),
+				AdditionalDirectories: []AdditionalDirectory{{pointer.String("/Volumes")}},
+			},
+			path: "/lima.yaml",
+			mockSvc: func(fs afero.Fs, l *mocks.Logger) {
+				err := afero.WriteFile(fs, "/lima.yaml", []byte("memory: 4GiB\ncpus: 8"), 0o600)
+				require.NoError(t, err)
+			},
+			postRunCheck: func(t *testing.T, fs afero.Fs) {
+				buf, err := afero.ReadFile(fs, "/lima.yaml")
+				require.NoError(t, err)
+
+				// limayaml.LimaYAML has a required "images" field which will also get marshaled
+				wantYaml := `images: []
```

Comment:
We can also get rid of the irrelevant `images: []`.

Reply:
image: [] is a required field so it will be anyways marshaled even if we don't input it.

Reply:
My bad, I meant right now we're asserting the whole file, so we need to mention it, but if we unmarshal the  bytes into a struct and only check the fields we care about like what we're doing in e2e tests, then we don't have to mention it and explain why it's there as it's irrelevant to us in the first place:

```go
			err = yaml.Unmarshal(cfgBuf, &limaCfg)
			gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
			gomega.Expect(*limaCfg.CPUs).Should(gomega.Equal(6))
			gomega.Expect(*limaCfg.Memory).Should(gomega.Equal("4GiB"))
```